src/EditorFeatures/CSharpTest/Diagnostics/Configuration/ConfigureSeverity/CodeStyleOptionBasedSeverityConfigurationTests.cs

```diff
@@ -1555,14 +1622,16 @@ public Customer()
     }
 }
         </Document>
-        <AnalyzerConfigDocument FilePath=""z:\\.editorconfig""></AnalyzerConfigDocument>
+        <AnalyzerConfigDocument FilePath=""z:\\.editorconfig"">[File.cs]
```

Comment:
More of a question rather then comment: Does this editorconfig header match a file named `File.cs` only in the current directory or all includes all files named `File.cs` in current directory as well as sub-directories?

Reply:
Confirmed by actually trying out an example, and your implementation is correct - it is the latter.